NPM `build` task is broken on MacOS M1

### Description

When doing an `npm run build` on main branch get then following error:

```
npm run build                                                                                                                                           ✔  37s  
> build
> npx lerna run build

lerna notice cli v7.1.4
lerna ERR! Error: Cannot find module '@nx/nx-darwin-arm64'
lerna ERR! Require stack:
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/native/index.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/project-graph/utils/retrieve-workspace-files.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/utils/nx-plugin.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/config/workspaces.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/config/configuration.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/daemon/client/client.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/lerna/dist/index.js
lerna ERR! - /Users/user/Projects/hedera-json-rpc-relay/node_modules/lerna/dist/cli.js
lerna ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1039:15)
lerna ERR!     at Module._load (node:internal/modules/cjs/loader:885:27)
lerna ERR!     at Module.require (node:internal/modules/cjs/loader:1105:19)
lerna ERR!     at require (node:internal/modules/cjs/helpers:103:18)
lerna ERR!     at Object.<anonymous> (/Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/native/index.js:135:29)
lerna ERR!     at Module._compile (node:internal/modules/cjs/loader:1218:14)
lerna ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
lerna ERR!     at Module.load (node:internal/modules/cjs/loader:1081:32)
lerna ERR!     at Module._load (node:internal/modules/cjs/loader:922:12)
lerna ERR! lerna Cannot find module '@nx/nx-darwin-arm64'
lerna ERR! lerna Require stack:
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/native/index.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/project-graph/utils/retrieve-workspace-files.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/utils/nx-plugin.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/config/workspaces.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/config/configuration.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/nx/src/daemon/client/client.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/lerna/dist/index.js
lerna ERR! lerna - /Users/user/Projects/hedera-json-rpc-relay/node_modules/lerna/dist/cli.js

```

I belive this was introduced after PR: https://github.com/hashgraph/hedera-json-rpc-relay/pull/1314

### Steps to reproduce

```

git clone https://github.com/hashgraph/hedera-json-rpc-relay.git
git checkout main
npm install
npm run build

```

### Additional context

_No response_

### Hedera network

other

### Version

main

### Operating system

macOS

--- a/package.json
+++ b/package.json
@@ -68,6 +68,7 @@
     },
     "overrides": {
         "protobufjs": "^7.2.4",
-        "semver": "^7.5.3"
+        "semver": "^7.5.3",
+        "nx": "16.5.5"
     }
 }

